hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3946,3 +3950,30 @@ func (r *HostedClusterReconciler) reconcileAWSSubnets(ctx context.Context, creat
 	}
 	return nil
 }
+
+// defaultAPIPortIfNeeded defaults the apiserver port on Azure management clusters as a workaround
+// for https://bugzilla.redhat.com/show_bug.cgi?id=2060650: Azure LBs with port 6443 don't work
```

Comment:
Is this because of a clash with the management cluster?

Reply:
It seems so but I am not really sure I understand why